README.md

```diff
@@ -212,25 +212,25 @@ You can stress-test a server by:
 
 ## Contributors
 
-Allen Rohner       
-Andre Branco       
-Andy Fingerhut       
-Christoph Henkelmann       
-Colin Jones       
-Dante Briones      
-Dimas Guardado       
-Gerrit Hentschel       
-J. David Lowe       
-Jeff Palmucci       
-Joe Gallo       
-Kevin DeJong       
-Kyle Kingsbury      
-Paul Biggar       
-Paul Stadig       
-Phil Hagelberg       
-Roman Flammer       
-Sam Ritchie       
-Stephen Nelson         
+Allen Rohner
+Andre Branco
+Andy Fingerhut
+Christoph Henkelmann
+Colin Jones
+Dante Briones
+Dimas Guardado
+Gerrit Hentschel
+J. David Lowe
+Jeff Palmucci
+Joe Gallo
+Kevin DeJong
+Kyle Kingsbury
+Paul Biggar
+Paul Stadig
+Phil Hagelberg
+Roman Flammer
+Sam Ritchie
+Stephen Nelson
 
```

Comment:
Note you removed all the newlines from these (or, likely your editor did).

Reply:
Oops. Yes, I have configured emacs to remove trailng spaces. Sorry for not reviewing the commit more thoroughly.